lib/rubocop/cop/lint/redundant_safe_navigation.rb

```diff
@@ -6,7 +6,7 @@ module Lint
       # Checks for redundant safe navigation calls.
       # Use cases where a constant, named in camel case for classes and modules is `nil` are rare,
       # and an offense is not detected when the receiver is a constant. The detection also applies
-      # to literal receivers, except for `nil`.
+      # to `self`, and literal receivers, except for `nil`.
```

Comment:
That's my two cents. In this case, the comma seems unnecessary.

```suggestion
      # to `self` and literal receivers, except for `nil`.
```

Reply:
Because there are two separate clauses that detection applies to (1. `self`; 2. literal receivers, except for `nil`), the comma is useful to emphasize that the "except for" clause only belongs to the second one. However, I've changed it slightly to further emphasize that (by adding a second "to"), thanks.